MolecularNodes/obj.py

```diff
@@ -45,6 +45,7 @@ def create_object(name: str, collection: bpy.types.Collection, locations, bonds=
     mol_mesh = bpy.data.meshes.new(name)
     mol_mesh.from_pydata(locations, bonds, faces=[])
     mol_object = bpy.data.objects.new(name, mol_mesh)
+    mol_object['type'] = 'molecule'
```

Comment:
This is something I have been thinking about for a while. I haven't come up with any kind of scheme, but the different objects that are added to Blender though Molecular Nodes currently aren't tagged with any identifying information. Tagging with `molecule` and maybe other info is a good idea. Open to suggestions for naming schemes.